recipes/filemagic/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "filemagic" %}
+{% set version = "1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "e684359ef40820fe406f0ebc5bf8a78f89717bdb7fed688af68082d991d6dbf3"
+
+build:
+  number: 0
+  skip: True  # [not linux]
```

Comment:
It seems like this actually a noarch package, when I checked on PyPI

Reply:
Thanks for the review @scopatz

Actually the package is pure Python but it is a wrapper around `libmagic` that is available only on unix system. I modified the recipe to `noarch` and add the dependency on `libmagic`. Is it a better recipe?